### DISCUSSION TITLE
How to ACK a command without responding?

### DISCUSSION THREAD

**Discussion by isiah-lloyd** [discordjs/discord.js#6907] [Q&A] [answered]
If you don't respond to an slash command interaction, Discord will show the user that the command failed. In the Discord docs, they state that there is a `PONG` interaction callback type. [https://discord.com/developers/docs/interactions/receiving-and-responding#interaction-response-object-interaction-callback-type]. Could this be implemented so we can use commands without having to use .reply()?

**Comment by almostSouji** [comment, marked as answer]
There is no "acknowledge without action" (anymore). If the user interacts with a bot (by pressing a button, using a dropdown, etc.) they generally expect something to happen, be that an update of the message the button is on, or an (ephemeral) reply.

The `PONG` interaction callback is for `PING` payloads that discord sends to interaction-only webhook applications. This is detailed in the documentation for this type of receiving interactions.: https://discord.com/developers/docs/interactions/receiving-and-responding#receiving-an-interaction

>     These steps are only necessary for webhook-based Interactions. It is not required for receiving them over the gateway.
>  Your endpoint must be prepared to ACK a PING message
    Your endpoint must be set up to properly handle signature headers--more on that in Security and Authorization
    
Discord.js only supports receiving interactions through gateway events (which is mutually exclusive with the above-mentioned webhook callbacks).
